src/app/page.tsx

```diff
@@ -20,10 +26,7 @@ const Home: FC = () => {
   });
 
   const [open, setOpen] = useState(false);
-
-  console.log('base: ', value.base);
-  console.log('password: ', value.password);
-  console.log('phoneNumber: ', value.phoneNumber);
+  const [modal, setModal] = useQueryState(MODAL_QUERY_STATE);
```

Comment:
```suggestion
  const [, setModal] = useQueryState(MODAL_QUERY_STATE);
```

Если тебе не нужна само состояние, а только функция для установки - можно вот так писать, чтобы линтер не ругался